test/local.tests.ts

```diff
@@ -280,5 +280,31 @@ int Add(int x, int y)
                 Token.Punctuation.CloseBrace
             ]);
         });
+
+        it("local function with attribute (issue #304)", async () => {
+            const input = Input.InMethod(`
+[Attribute]
```

Comment:
Man, I do _not_ know what we're going to do for collection expressions here. It's not going to be fun.